doc/src/devdocs/inference.md

```diff
@@ -109,26 +108,11 @@ cst = map(cost, ci.code)
 
 # output
 
-31-element Array{Int64,1}:
```

Comment:
Why change this?

Reply:
If you run the demo, you get only a 5-element array. Try running this in `REPL`: 

```julia
# A demo on `fill(3.5, (2, 3))
f = fill
tt = Tuple{Float64, Tuple{Int,Int}}
# Create the objects we need to interact with the compiler
params = Core.Compiler.Params(typemax(UInt))
mi = Base.method_instances(f, tt)[1]
ci = code_typed(f, tt)[1][1]
opt = Core.Compiler.OptimizationState(mi, params)
# Calculate cost of each statement
cost(stmt::Expr) = Core.Compiler.statement_cost(stmt, -1, ci, opt.sptypes, opt.slottypes, opt.params)
cost(stmt) = 0
cst = map(cost, ci.code)
```

The current documentation lists a five-element array, and I am not entirely sure why it has been changed to a 31-element array. If I am wrong, the second demo can be just ignored, but the `doctest` failed there as well due to a reason that I don't 100% understand.

Reply:
You might (likely) run it on a version of Julia that is different from the one where the doctests are run on (master).